mark_outliers_in_json.py

```diff
@@ -76,18 +76,26 @@ def get_outliers(all_outliers, window_size, threshold=1):
     return common, unique
 
 
-def get_all_outliers(data, window_size):
+def _tuckey_all_outliers(data, window_size):
+    # Ignore windows that do not have a full set of data.
     all_outliers = list()
     for index, datum in enumerate(data):
         l_slice, r_slice = _clamp_window_size(index, len(data), window_size)
+        if (r_slice - l_slice) < window_size and l_slice == 0:
```

Comment:
I don't think so. `r_slice` and `l_slice` are array indices, so we want to know whether they are a window size apart. So, this is saying "have we failed to fill a window on the left hand side of the chart"